embedchain/loaders/excel.py

```diff
@@ -0,0 +1,79 @@
+from embedchain.loaders.base_loader import BaseLoader
+import pandas as pd
+from io import StringIO
+import os
+import hashlib
+
+
+class ExcelLoader(BaseLoader):
+    @staticmethod
+    def _convert_sheet_to_csv(file: str, sheet_name: str):
+        """
+        file: str
+            Excel file location
+        sheet_name: str
+            Name of the Excel sheet
+        """
+
+        # Read the specified sheet from the Excel file
+        temp_data = pd.read_excel(file, sheet_name=sheet_name)
+
+        # Create buffer
+        buffer = StringIO()
+
+        # Convert to CSV and write to buffer
+        temp_data.to_csv(buffer, index=False)
+
+        # Get the CSV data as a string
+        csv_string = buffer.getvalue()
+
+        # Clean string
+        csv_string = csv_string.replace("\r", "")
+
+        csv_string = csv_string.strip("\n")
```

Comment:
It will be better if each row is in new line in the CSV. Please dont strip the newline character `\n`.

Reply:
Stripping the newline character helps to maintain value obtained from buffer, which always ends with `\n`.

Reply:
Sounds good!